### ISSUE TITLE
[MU4 Issue] "Staff/System text properties" (context menu): change to "Playback properties"

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#13382]
**Describe the issue**

The current wording of the options "Staff/System text properties" in the context menus for Staff/System text is ambiguous. It looks as if the option is about "text properties" whereas in reality it is about playback properties.

**Solution?**

I would like to suggest changing the wording of "Staff text properties" and "System text properties" to "Playback properties".

**Comment by MarcSabatella** [comment]
I agree this is misleading,  The thing is, it's really "Staff/System text - properties", not "Staff/System - text properties".  That is, "properties of staff or system text", not "text properties of the staff or system".  But the capitalization definitely makes it seem like the latter, and it was somewhat less confusing in the past when it used title case - "Staff/System Text Properties".

Ultimately, there's no reason this dialog should exist in the future; I would expect these settings to eventually make their way to the main Properties panel, under the Playback button.
